modules/network-observability-flowmetrics-charts.adoc

```diff
@@ -0,0 +1,102 @@
+    unit: Bps
+    type: SingleStat
+    queries:
+    - promQL: "sum(rate($METRIC[2m]))"
+      legend: ""
+  - dashboardName: Main
+    sectionName: External
+    title: Top external ingress traffic per workload
+    unit: Bps
+    type: StackArea
+    queries:
+    - promQL: "sum(rate($METRIC{DstK8S_Namespace!=\"\"}[2m])) by (DstK8S_Namespace, DstK8S_OwnerName)"
+      legend: "{{DstK8S_Namespace}} / {{DstK8S_OwnerName}}"
+# ...
+----
+<1> The same namespace, netobserv, is used across all components. You can optionally use a different namespace for the different components. It is important to keep namespace names consistent across all components.
+
+This example creates two panels:
+
+* A textual "single stat" showing the global external ingress rate summed across all dimensions
+* A timeseries graph showing the same metric per destination workload
+
+For more information about the query language, refer to the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus documentation].
+====
+
+.Chart for RTT latency for cluster external ingress traffic
+[%collapsible]
+====
+[source,yaml]
+----
+# ...
+  charts:
+  - dashboardName: Main
+    title: External ingress TCP latency
+    unit: seconds
+    type: SingleStat
+    queries:
+    - promQL: "histogram_quantile(0.99, sum(rate($METRIC_bucket[2m])) by (le)) > 0"
+      legend: "p99"
+  - dashboardName: Main
+    sectionName: External
+    title: "Top external ingress sRTT per workload, p50 (ms)"
+    unit: seconds
+    type: Line
+    queries:
+    - promQL: "histogram_quantile(0.5, sum(rate($METRIC_bucket{DstK8S_Namespace!=\"\"}[2m])) by (le,DstK8S_Namespace,DstK8S_OwnerName))*1000 > 0"
+      legend: "{{DstK8S_Namespace}} / {{DstK8S_OwnerName}}"
+  - dashboardName: Main
+    sectionName: External
+    title: "Top external ingress sRTT per workload, p99 (ms)"
+    unit: seconds
+    type: Line
+    queries:
+    - promQL: "histogram_quantile(0.99, sum(rate($METRIC_bucket{DstK8S_Namespace!=\"\"}[2m])) by (le,DstK8S_Namespace,DstK8S_OwnerName))*1000 > 0"
+      legend: "{{DstK8S_Namespace}} / {{DstK8S_OwnerName}}"
+# ...
+----
+
+This example uses the `histogram_quantile` function, to show p50 and p99.
+
+You might also want to show averages of histograms: this is done by dividing the metric `$METRIC_sum` by the metric `$METRIC_count`, which are automatically generated when you create an histogram. With the preceding example, the Prometheus query to do this would be as follows:
+
+[source,yaml]
+----
+promQL: "(sum(rate($METRIC_sum{DstK8S_Namespace!=\"\"}[2m])) by (DstK8S_Namespace,DstK8S_OwnerName) / sum(rate($METRIC_count{DstK8S_Namespace!=\"\"}[2m])) by (DstK8S_Namespace,DstK8S_OwnerName))*1000"
+----
+
+For more information about the query language, refer to the link:https://prometheus.io/docs/prometheus/latest/querying/basics/[Prometheus documentation].
+====
+
+.Verification
+. Once the pods refresh, navigate to *Observe* -> *Dashboards*.
```

Comment:
add a bullet to say "Search for NetObserv / Main dashboard" , we should add similar bullet as https://github.com/openshift/openshift-docs/pull/74412/files#diff-2a22749e0d0fdc529e24f45d6db04896a098ccb694cd48ed09a31709114bd1f4R35 to say dashboard Names are prefixed with "NetObserv / "